backend/server/routers/courses.py

```diff
@@ -117,6 +130,14 @@ def search(string):
     code_query = coursesCOL.find({"code": {"$regex": pat}})
     title_query = coursesCOL.find({"title": {"$regex": pat}})
 
+    years = sorted(ARCHIVED_YEARS, reverse=True)
+    if len(list(code_query.clone())) == 0 and len(list(title_query.clone())) == 0:
```

Comment:
wait i missed this - we want to include this query in all cases, and indicate that it is legacy. this means that we should be returning an 'is_legacy' for each course.